src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -249,10 +264,21 @@ export const canShow = async (
 		hasRequiredConsents(),
 	]);
 
+	await maybeWipeUserData(apiKey as string, brazeUuid);
```

Comment:
Similarly, looking around this function I can see that we're fetching and using the consent that is determined for Braze, which looks just right. However, I want to check whether there are cases where a lack of consent means none of this code will execute? If no consents are given will the commercial bundle not load any of the Braze logic, or is it separately loaded and fully taking its own responsibility for consent?

Reply:
My understanding is that this will be loaded either way so I don't think there are cases where none of this code will load because of consent.